release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -274,10 +273,86 @@ deprecated in {product-title} 3.11 and has been removed. You must use the
+
+[IMPORTANT]
+====
+Service Catalog is not installed by default in {product-title} 4, however it now
+blocks upgrades to {product-title} 4.5 if installed.
+====
+
+Service Catalog, Template Service Broker, Ansible Service Broker, and their
+associated Operators were deprecated starting in {product-title} 4.2. Ansible
+Service Broker, including Ansible Service Broker Operator and related APIs and
+APBs, were removed in {product-title} 4.4.
+
+Service Catalog, Template Service Broker, and their associated Operators are now
+removed in {product-title} 4.5, including the following related API:
+
+* `.servicecatalog.k8s.io/v1beta1`
+
+[NOTE]
+====
+Templates are still available in {product-title} 4.5, but they are no longer
+handled by Template Service Broker. By default, the Samples Operator handles Red
+Hat Enterprise Linux (RHEL)-based {product-title} ImageStreams and Templates.
+See
+xref:../openshift_images/configuring-samples-operator.adoc#configuring-samples-operator[Configuring the Samples Operator]
+for details.
+====
+
+The `service-catalog-controller-manager` and `service-catalog-apiserver` cluster
+Operators were set to `Upgradeable=false` in 4.4. This means that they block
+cluster upgrades to the next minor version, 4.5 in this case, if they are still
+installed at that time. Upgrading to z-stream releases such as 4.4.z, however,
+are still permitted in this state.
+
+If Service Catalog and Template Service Broker are enabled in 4.4, specifically
+if their management state is set to `Managed`, the web console warns cluster
+administrators that these features are still enabled. The following alerts can
+be viewed from the *Monitoring* -> *Alerting* page on a 4.4 cluster and have a
+*Warning* severity:
+
+* `ServiceCatalogAPIServerEnabled`
+* `ServiceCatalogControllerManagerEnabled`
+* `TemplateServiceBrokerEnabled`
+
+If they are still enabled on a 4.4 cluster, cluster administrators can see
+link:https://docs.openshift.com/container-platform/4.4/applications/service_brokers/uninstalling-service-catalog.html#sb-uninstalling-service-catalog[Uninstalling Service Catalog] and link:https://docs.openshift.com/container-platform/4.4/applications/service_brokers/uninstalling-template-service-broker.html#sb-uninstalling-template-service-broker[Uninstalling Template Service Broker]
+in the {product-title} 4.4 documentation to uninstall it, which permits cluster
+upgrades to 4.5.
+
+In 4.5, a pair of Jobs are created in a new `openshift-service-catalog-removed`
+namespace to run during the cluster upgrade process. Their behavior depends on
+the management state of Service Catalog:
+
+* `Removed`: The Jobs remove the following Service Catalog items:
+** Operators
+** namespaces
+** Custom Resources (CRs)
+** ClusterRoles
+** ClusterRoleBindings
+
+* `Unmanaged`: The Jobs skip removal and do nothing.
+
+* `Managed`: The Jobs report an error in logs. This should not occur as upgrades
+would be blocked, but the Jobs do not do anything just in case.
```

Comment:
I'm a little confused in this bullet:
- By "this should not occur", do you mean an error or the `Unmanaged` state? 
- "Just in case" _what_? 

Maybe a slight rewording, if the meaning still holds. Something like:

> This state (?) is unlikely to occur because upgrades would be blocked. The Jobs do nothing.